prinz/src/main/scala/pl/touk/nussknacker/prinz/model/ModelInstance.scala

```diff
@@ -6,7 +6,7 @@ import scala.concurrent.Future
 
 abstract class ModelInstance(model: Model, protected val signatureProvider: SignatureProvider) {
 
-  type ModelRunResult = Future[Either[ModelRunException, Map[String, _]]]
+  type ModelRunResult = Future[Either[ModelRunException, java.util.Map[String, _]]]
```

Comment:
`Map` could be imported.

Reply:
scala Mp is automatically imported so there are conflicting names - I imported java Map as JMap and no there is no java.util prefix in code needed